chemprop/data/utils.py

```diff
@@ -656,7 +656,46 @@ def get_inequality_targets(path: str, target_columns: List[str] = None) -> List[
                 raise ValueError(f'A target value in csv file {path} contains both ">" and "<" symbols. Inequality targets must be on one edge and not express a range.')
 
     return gt_targets, lt_targets
+def weight_split(data: MoleculeDataset,
+                 bias: str,
```

Comment:
I don't think `bias` is a very descriptive argument name? Maybe something like `train_on_small`?

Reply:
I think the only reasonable way is to put the small ones in the training, so I removed this arg altogether. Thanks for all your comments.